apps/search/style/search.css

```diff
@@ -291,3 +290,12 @@ gaia-grid {
 [data-loading=true] .loading {
   display: block;
 }
+
+html[dir="rtl"] #suggestions-provider {
+  padding: 0 1rem 0 2rem;
```

Comment:
I believe the convention for padding has been to use moz-padding-start/moz-padding-end, no?